pallets/ethereum-events/src/mock.rs

```diff
@@ -674,22 +670,25 @@ impl ExtBuilder {
     #[allow(dead_code)]
     pub fn with_genesis_config(mut self) -> Self {
         let _ = pallet_ethereum_events::GenesisConfig::<TestRuntime> {
-            validator_manager_contract_address: H160::from(VALIDATORS_MANAGER_CONTRACT),
-            lifting_contract_address: H160::from(LIFTING_CONTRACT),
             nft_t1_contracts: vec![(H160::from(NFT_CONTRACT), ())],
             processed_events: vec![],
             lift_tx_hashes: vec![],
             quorum_factor: QUORUM_FACTOR,
             event_challenge_period: EVENT_CHALLENGE_PERIOD,
         }
         .assimilate_storage(&mut self.storage);
+
+        let _ = pallet_avn::GenesisConfig::<TestRuntime> {
```

Comment:
is `with_genesis_config` run for all test configs? as is, it will be added in the genesis for the tests that use that, and not for example the `initial_lifts` one.